buildscripts/build.sh

```diff
@@ -110,7 +110,8 @@ fi
 env GOOS=$GOOS GOARCH=$GOARCH go build ${BUILD_TAG} -ldflags \
     "-X github.com/openebs/maya/pkg/version.GitCommit=${GIT_COMMIT} \
     -X main.CtlName='${CTLNAME}' \
-    -X github.com/openebs/maya/pkg/version.Version=${VERSION}" \
+    -X github.com/openebs/maya/pkg/version.Version=${VERSION} \
+    -X github.com/openebs/dynamic-nfs-provisioner/provisioner.NFSServerDefaultImage=${NFSSERVERIMG}" \
```

Comment:
q_: so will this replace the variable/constant (`NFSServerDefaultImage`) value set in code from `openebs/nfs-server-alpine:latest` to current built image?

Reply:
`:latest` tag exists for both the image. `openebs/nfs-server-alpine:latest` will be used only if binaries are built through go cli, instead of using makefile. I'm ok if we remove this default value. @niladrih what do you think?

Reply:
Yes. Same with me. I hadn't noticed the default set value. This can be removed.